### DISCUSSION TITLE
Which chromeOS recovery is used for 12th gen intel cpus?

### DISCUSSION THREAD

**Discussion by bdodroid** [sebanc/brunch#1651] [Q&A] [answered]
With the release of a couple 12th gen intel cpu chromebooks like HP's Dragonfly series, I was wondering what chromeOS "codename" they are using?

Or has anyone found a version that works well with 12th gen?

Mainly asking because I'm getting display issues using "Volteer" on a couple of 12th gen pcs and I want to try some other versions.

**Comment by bdodroid** [comment, marked as answer]
If anyone is curious it looks like 12th gen intel chromebooks are using the BRYA builds.

I just realize you could search by chromebook model in the Chromeium Dash to see what was being used.....
https://chromiumdash.appspot.com/serving-builds?deviceCategory=Chrome%20OS

**Reply by videeshhh** [reply]
What build I should use for i5 12400 16 gb ram ? Pls help no build is working its stuck on ChromeOS logo pls help me

**Comment by ycavan** [comment]
Brya is the proper recovery image to use.  Here's some stuff that will help you getting it running properly.

1. If iwlwifi isn't loading, you'll need to compile and install backports_iwlwifi.  I used a netgear usb device in the interim for wifi: 0846:9055 NetGear, Inc. A6150.  After that, I installed Brioche and set up an arch linux container. From here, install base-devel and git.  Clone the backports_iwlwifi repository and compile it.  sudo make install and reboot to get your intel wifi working.
2. NVidia devices are disabled by default.  If you have an NVidia gpu and want to use it, edit /lib/udev/rules.d/99-nvidia_pm.rules to comment out all of the lines. ( put a # in front of the non-#'d lines ) and reboot.  You'll see your NVidia gpu in lspci again.
3. IF you want to use your NVidia gpu in crouton, you can download the latest NVidia drivers to match your crouton distro's supported version.  Start up your arch container ( from step 1 ) and sudo sh the .run file to install the drivers.  No reboot necessary.  IF you want i386 support, you should make sure to edit /etc/pacman.conf and uncomment out the [multilib] section.  Do a sudo pacman -Syu to make sure it's all updated and then sudo sh the nvidia .run file.
4. IF you have an NVidia gpu that you wanted to pass to qemu, you can modify the linux cmd line by creating a mount point: sudo mkdir /mnt/efi and mount the 12th partition.  For me that's /dev/nvme1n1p12.  After that it's simple enough to sudo nano /mnt/efi/efi/boot/settings.cfg and append intel_iommu=on iommu=pt pci-stub.ids=nvidia_audio_pci_id vfio-pci.ids=nvidia_gpu_pci_id,nvidia_audio_pci_id to the command line string.  Save it, unmount /mnt/efi and reboot.  After the reboot you can then sudo modprobe vfio-pci and if you do a sudo lspci -nnk you should see that pci device 0000:0001.0 is using vfio-pci.  Now you can pass it to your qemu vm.

**Reply by videeshhh** [reply]
What build I should use for i5 12400 16 gb ram ? Pls help no build is working its stuck on ChromeOS logo pls help me

**Comment by ycavan** [comment]
First, what's the model of the machine you're trying to run brunch on?  Did you use Brya as your build? When you boot up, can you enter the configuration and set it to have no boot screen ( in order to see where you're stuck? )
    On Sunday, January 8, 2023 at 09:00:14 AM CST, Vaman223 ***@***.***> wrote:  
 
 


What build I should use for i5 12400 16 gb ram ? Pls help no build is working its stuck on ChromeOS logo pls help me

**Comment by OurCodeBase** [comment]
Please anyone tell me about codename for chrome os for my HP i3 1215U processor.

**Reply by zenith246** [reply]
Hey did you find any way I also have Hp15s i3 1215u processor
